compiler/rustc_trait_selection/src/error_reporting/traits/ambiguity.rs

```diff
@@ -610,6 +610,20 @@ impl<'a, 'tcx> TypeErrCtxt<'a, 'tcx> {
                 )
                 .with_span_label(span, format!("cannot normalize `{alias}`"))
             }
+            ty::PredicateKind::Clause(ty::ClauseKind::UnstableFeature(sym)) => {
+                if let Some(e) = self.tainted_by_errors() {
+                    return e;
+                }
+
+                let mut err = self.dcx().struct_span_err(
+                    span,
+                    format!("unstable feature `{sym}` is used without being enabled."),
+                );
+                err.help(format!("The feature can be enabled through #[unstable_feature_bound({sym})] in std/core, or \
+                                #[feature({sym})] outside of std/core."
```

Comment:
```suggestion
                err.help(format!("The feature can be enabled through #[unstable_feature_bound({sym})] in std/core, or \
                                #[feature({sym})] outside of std/core."
```
I think we should check if `staged_api` is enabled, if it is we should say:
``help: The feature can be enabled by marking the current item with `#[unstable_feature_bound({sym})` ``
And if it's not then we should just tell the user to add `feature` to the crate root without mentioning `unstable_feature_bound` or `std/core` whatsoever

Reply:
Talking about std/core or `unstable_feature_bound` will be confusing for users as most people are not in std/core and `unstable_feature_bound` is not useable :D